solidity-v1/dashboard/src/lib/keep/index.js

```diff
@@ -41,7 +44,9 @@ class Keep {
   static initialize(web3) {
     const keep = new Keep(web3)
     keep.initializeContracts()
-    keep.initializeServices()
+    keep._initializePREContract(web3).then(() => {
+      keep.initializeServices()
+    })
```

Comment:
If you agree with the above comments we can leave as it was:
```
keep.initializeContracts()
keep.initializeServices()
```

Reply:
a3ab04267c9d6acc69725ef3e469c8817810ac8e